pagerduty/resource_pagerduty_event_orchestration_path_unrouted.go

```diff
@@ -203,10 +203,31 @@ func resourcePagerDutyEventOrchestrationPathUnroutedCreate(ctx context.Context,
 
 // EventOrchestrationPath cannot be deleted, use update to add / edit / remove rules and sets
 func resourcePagerDutyEventOrchestrationPathUnroutedDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	var diags diag.Diagnostics
+	client, err := meta.(*Config).Client()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	// In order to delete an Unrouted Orchestration an empty orchestration path
+	// config should be sent as an update.
+	emptyPath := emptyOrchestrationPathStructBuilder("unrouted")
+	routerID := d.Get("event_orchestration").(string)
```

Comment:
Did you mean to name the variable `unroutedID` or `orchestrationID` here?

Reply:
I should have mean `orchestrationID` hehe 😅